social-media-guide.md

```diff
@@ -0,0 +1,65 @@
+# Social Media Guide
+
+This document is a loose set of policies and guidelines for social media usage
+by the OpenTelemetry project. It is not intended to be an exhaustive list of
+what and how we should post or market the project, but more a set of principles
+and values that we should follow in order to maintain consistency and
+professionalism.
+
+## Our Goals
+
+OpenTelemetry's use of social media (Twitter, LinkedIn, YouTube, Mastodon, etc.)
+is intended to help us achieve the following goals:
+
+- Increase awareness of the project.
+- Highlight the work of contributors and maintainers.
+- Help educate and inform users and potential users of the project.
+
+## Our Values
+
+OpenTelemetry is a community-driven project, and as such, our social media usage
+should reflect our community standards. Please familiarize yourself with our
+[mission, vision, and values](./mission-vision-values.md) as well as our [code
+of conduct](./code-of-conduct.md) as the primary reference for how we should
+represent ourselves and conduct ourselves on social media.
+
+## Guidelines for Social Media Usage
+
+When creating social content, keep the following general guidelines in mind:
+
+- Be professional and courteous.
+  - Avoid hyperbole and exaggeration, or "extremely online" references and
+  behavior.
+- Do not use OpenTelemetry social accounts to promote a specific proprietary
```

Comment:
Also do not use social accounts to disparage or denigrate the same things

Reply:
Could you clarify? "Don't use otel social accounts to disparage or denigrate other products or projects"?